applications/nrf5340_audio/src/utils/board_init.c

```diff
@@ -121,19 +125,20 @@ int nrf5340_audio_dk_init(void)
 		}
 	}
 
-	ret = leds_set();
-	if (ret) {
-		LOG_ERR("Failed to set LEDs");
-		return ret;
-	}
-
 	if (IS_ENABLED(CONFIG_SD_CARD_PLAYBACK)) {
 		ret = sd_card_playback_init();
 		if (ret) {
 			LOG_ERR("Failed to initialize SD card playback");
 			return ret;
 		}
 	}
+#endif /* CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP */
```

Comment:
Discuss: We want to perform the LED init as early as possible.

Reply:
The LEDs are the first item initialized here?
It would have to be in the main for each use case to move it higher!